ci-operator/config/stolostron/search-indexer/stolostron-search-indexer-main.yaml

```diff
@@ -0,0 +1,79 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.17-linux:
+    name: builder
+    namespace: open-cluster-management
```

Comment:
Should this also be `namespace: stolostron`

Reply:
Yes, the builder image `stolostron_builder_go1.17-linux` is in namespace `stolostron`